src/stars-block.css

```diff
@@ -0,0 +1,40 @@
+.stars_block {
+  flex-direction: row;
+  display: flex;
+  background: #fff;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  margin-top: 110px;
+}
+  
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  display: flex;
+  padding-left: 4px;
+}
+
+.stars--4 > :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
you must move all styles from previous task

Reply:
not fixed